identifiers/country-kr.csv

```diff
@@ -17,6 +17,13 @@ ocd-division/country:kr/ed:bucheon_city_1,Bucheon City First
 ocd-division/country:kr/ed:bucheon_city_2,Bucheon City Second
 ocd-division/country:kr/ed:bucheon_city_3,Bucheon City Third
 ocd-division/country:kr/ed:bucheon_city_fourth,Bucheon City Fourth
+ocd-division/country:kr/ed:buk_district,Buk District
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_2,Buk District and Gangseo District Second
+ocd-division/country:kr/ed:bukd_district_1,Buk District First Daegu

```

Comment:
Can we make these ids a bit more descriptive by including the full city name? Keeping it consistent with other ids such as `ocd-division/country:kr/ed:anyang_city_dongan_district_1`.

```suggestion
ocd-division/country:kr/ed:daegu_city_buk_district_1,Buk District First Daegu
```